sdk/identity/identity/samples/typescript/src/interactiveBrowserCredential.ts

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { InteractiveBrowserCredential } from "@azure/identity";
+import { KeyClient } from "@azure/keyvault-keys";
+
+async function main(): Promise<void> {
+  const credential = new InteractiveBrowserCredential({
```

Comment:
Is this meant to be run in a browser or will it open a browser for me if I run it in Node?

Reply:
Let me write you about this, since I'm wondering to which extent we will be able to provide automation for these samples.

Reply:
I'll drop the Interactive Browser Credential and the Device Code Credential samples. The reasoning is:

- We have a markdown file at the root of the samples folder that shows how to work with these.
- These won't be able to work on CI because they require user input.